include/internal/dso.h

```diff
@@ -163,71 +164,8 @@ DSO *DSO_dsobyaddr(void *addr, int flags);
  */
 void *DSO_global_lookup(const char *name);
 
-/* BEGIN ERROR CODES */
-/*
- * The following lines are auto generated by the script mkerr.pl. Any changes
- * made after this point may be overwritten when the script is next run.
- */
-
 int ERR_load_DSO_strings(void);
```

Comment:
Huh...  I just noticed this, and I wonder, why do you keep it around here?  It's declared in `dsoerr.h` anyway...

This goes for all headers you've modified similarly...